### PR TITLE
Update localstack and opensearch

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

**Comment by iHiD** [comment]
@ErikSchierboom Can you check you're happy with this please. `_type` was removed: https://github.com/opensearch-project/OpenSearch/issues/2979

File: Gemfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ gem 'kaminari'
 gem 'oj', '~> 3.14.0'
 
 # Setup dependencies
-gem 'exercism-config', '>= 0.106.0'
+gem 'exercism-config', '>= 0.107.0.beta1'
```
Comment (iHiD) [inline Gemfile]: We need to release an actual config gem and update this before merging,

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ We use localstack and opensearch, and run them via Docker.
 
 ```bash
 docker run -dp 3042:8080 -p 3040:4566 -p 3041:4566 localstack/localstack
-docker run -dp 9200:9200 -e "discovery.type=single-node" opensearchproject/opensearch:1.1.0
+docker run -dp 9200:9200 -e "discovery.type=single-node" opensearchproject/opensearch:2.9.0
```
Comment (iHiD) [inline README.md:72]: Maybe we save these versions somewhere so that they're consistent in dev + ci?

Comment (ErikSchierboom) [reply README.md:72]: Done in https://github.com/exercism/website/pull/5824/commits/57ac597e9de284a808ec2009971ad1a7a4290590

Looks to be working: https://github.com/exercism/website/actions/runs/5724277008/job/15510467895

File: bin/dev [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-docker run -dp 3042:8080 -p 3040:4566 -p 3041:4566 localstack/localstack:2.2.0
-docker run -dp 9200:9200 -e "discovery.type=single-node" opensearchproject/opensearch:2.9.0
+docker run -dp 3042:8080 -p 3040:4566 -p 3041:4566 $(jq -r '.localstack' .dockerimages.json)
+docker run -dp 9200:9200 -e "discovery.type=single-node" $(jq -r '.opensearch' .dockerimages.json)
```
Comment (iHiD) [inline bin/dev:4]: Neat!

**Pull request merged**